security/pfSense-pkg-acme/files/usr/local/www/acme/acme_certificates_edit.php

```diff
@@ -0,0 +1,432 @@
+
+// 'processing' done, make all simple fields usable in html.
+foreach($simplefields as $field){
+	$pconfig[$field] = htmlspecialchars($pconfig[$field]);
+}
+
+?>
+<script type="text/javascript">
+	function clearcombo(){
+	  for (var i=document.iform.serversSelect.options.length-1; i>=0; i--){
+		document.iform.serversSelect.options[i] = null;
+	  }
+	  document.iform.serversSelect.selectedIndex = -1;
+	}
+
+	function setCSSdisplay(cssID, display)
+	{
+		var ss = document.styleSheets;
+		for (var i=0; i<ss.length; i++) {
+			var rules = ss[i].cssRules || ss[i].rules;
+			for (var j=0; j<rules.length; j++) {
+				if (rules[j].selectorText === cssID) {
+					rules[j].style.display = display ? "" : "none";
+				}
+			}
+		}
+	}
+	function toggleCSSdisplay(cssID)
+	{
+		var ss = document.styleSheets;
+		for (var i=0; i<ss.length; i++) {
+			var rules = ss[i].cssRules || ss[i].rules;
+			for (var j=0; j<rules.length; j++) {
+				if (rules[j].selectorText === cssID) {
+					rules[j].style.display = rules[j].style.display === "none" ? "" : "none";
+				}
+			}
+		}
+	}
+	
+	function updatevisibility()
+	{
+		d = document;
+		// IE needs components found into javascript variables
+	}
+</script>
+</head>
+<?php
+if (isset($input_errors)) {
+	print_input_errors($input_errors);
+}
+
+$counter=0;
+
+$form = new \Form;
+
+$section = new \Form_Section('Edit Certificate options');
+$section->addInput(new \Form_Input('name', 'Name', 'text', $pconfig['name']
+))->setHelp('The name set here will also be used to create or overwrite a certificate that might already exist with this name in the pfSense certificate m.');
+$section->addInput(new \Form_Input('desc', 'Description', 'text', $pconfig['desc']));
+$activedisable = array();
+$activedisable['active'] = "Active";
+$activedisable['disabled'] = "Disabled";
+$section->addInput(new \Form_Select(
+	'status',
+	'Status',
+	$pconfig['status'],
+	$activedisable
+));
+$a_accountkeys = &$config['installedpackages']['acme']['accountkeys']['item'];
+$section->addInput(new \Form_Select(
+	'acmeaccount',
+	'Acme Account',
+	$pconfig['acmeaccount'],
+	form_name_array($a_accountkeys)
+));
+
+$a_accountkeys = &$config['installedpackages']['acme']['accountkeys']['item'];
+$section->addInput(new \Form_Select(
+	'keylength',
+	'Key Size',
+	$pconfig['keylength'],
+	form_name_array($a_keylength)
+));
+
+$section->addInput(new \Form_StaticText(
+	'Domain SAN list', 
+	"List all domain names that should be included in the certificate here, and how to validate ownership by use of a webroot or dns challenge<br/>"
+	. "Examples:<br/>"
+	. "/usr/local/www/.well-known/acme-challenge/<br/>"
+	. "/tmp/haproxy_chroot/.well-known/acme-challenge/"
+	. $domainslist->Draw($a_domains)
+));
+
+$section->addInput(new \Form_StaticText(
+	'Actions list', 
+	"Used to restart webserver processes this certificate has been renewed<br/>Examples:<br/>/etc/rc.restart_webgui<br/>/usr/local/etc/rc.d/haproxy.sh restart".
+	$actionslist->Draw($a_actions)
```

Comment:
This option uses the fa-level-down icon to create a new entry, which is unintuitive and easy to miss. Copy the style used by rowhelper in various areas of the GUI and use a button with text